### PR TITLE
NFT feedback

### PR DESCRIPTION
**Pull request by jordy25519**
Some early feedback

Adds:
- Set owner function to change collection owner
- Base metadata URI for a collection (https://eips.ethereum.org/EIPS/eip-1155#metadata)

Changes:
- `tokens/Tokens` renamed to `token_attributes/TokenAttributes`
- Move nft CollectionId to be defined in cennznet_primitives crate

Fixes:
- Update collectedTokens rpc for better api support

TODO:
- [ ] createTokenBatch() to mint X tokens with similar values
- [ ] balanceOf function (better to go on api side for now)
- [ ] Some way to prevent missing out on royalties when using 'transfer' function

### CODE REVIEW

**Comment by jordy25519** [comment]
run benchmark

File: cli/src/rpc.rs [hunk 1]
```diff
@@ -21,8 +21,8 @@
 
 use std::sync::Arc;
 
-use cennznet_primitives::types::{AccountId, AssetId, Balance, BlockNumber, Hash, Index};
-use cennznet_runtime::opaque::Block;
+use cennznet_primitives::types::{AccountId, AssetId, Balance, BlockNumber, Hash, Index, TokenId};
+use cennznet_runtime::{opaque::Block, CollectionId};
```
Comment (aliXsed) [inline cli/src/rpc.rs]: Should CollectionId come from `cennznet_primitives::types` the same as TokenId?

**Review by aliXsed — APPROVED** [review]
lgtm!

**Comment by jordy25519** [comment]
run benchmark

**Pull request merged**